aspnetcore/includes/bind7.md

```diff
@@ -13,7 +13,7 @@ Create the following `PositionOptions` class:
 
 An options class:
 
-* Must be non-abstract with a public parameterless constructor.
+* Must be non-abstract.
```

Comment:
I don't know what the correct grammatical term is for bullet points that complete the sentence started in the text above, but these bullet points have a mixture, e.g.
"An options class my be non-abstract" - (finishes the sentence)
"An options class all public read-write properties of the type are bound" (doesn't finish the sentence)

Our AI Overlords explain it like this:

> The bullet points that complete a sentence started in the text above them are typically referred to as "sentence fragments" or "fragmented bullet points." The introductory text sets the stage, and each bullet point serves as a continuation or completion of that thought, although not forming complete sentences on their own. This style is often used to create clear, concise lists while maintaining a grammatical connection to the introductory statement.

> In a more formal or academic setting, it might be referred to as "horizontal listing" where the introductory statement and the bullet points together form a grammatically complete structure. This term contrasts with "vertical listing" where each bullet point is a complete sentence.

> This structure enhances readability and helps to maintain a logical flow of information, particularly in business or technical writing where brevity and clarity are often valued.

So, it seems we're mixing horizontal and vertical listings.